create-a-theme-checklist.js

```diff
@@ -30,69 +30,71 @@ async function createMilestone() {
 					'Automatically generated milestone for ' + theme + '.',
 			}
 		);
-	} catch ( error ) {
-		console.error( '\x1b[41m', 'Milestone already created.' );
+	} catch (error) {
+		console.error('\x1b[41m', 'Milestone already created.');
 	}
 }
 
-async function createIssues( milestoneNumber ) {
+async function createIssues(milestoneNumber) {
 	const issues = [
-		'Create base theme from boilerplate',
+		'Create base theme',
 		'Create a demo site on dotcom',
 		'theme.json: Typography settings',
 		'theme.json: Color palette',
 		'theme.json: Alternative color palettes',
 		'theme.json: Margin/spacing settings + content layout',
 		'Templates: Page templates - Index',
+		'Templates: Page templates - Default',
```

Comment:
Why default and not just "page template" or "page"?

Reply:
Ok I see the other templates are called Page templates - 404 but those are not really pages, they are just 404 template or Search template instead.

Reply:
Yeah I know what you mean, I wasn't sure what to call this! I thought Page templates - Page sounded a little strange, but Default is maybe confusing. Maybe just 'Templates: page.html'?

Reply:
yeah, probably just naming templates what they are is the most straightforward you can get 🤣